sdk/search/Azure.Search.Documents/Directory.Build.props

```diff
@@ -7,8 +7,7 @@
 
     <!-- Use Azure.Core.Experimental's Spatial features -->
     <!-- (Note: a number of tests will need re-recording if you disable) -->
-    <UseAzureCoreExperimentalSpatial Condition="'$(UseAzureCoreExperimentalSpatial)' == ''">$(UseAzureCoreExperimental)</UseAzureCoreExperimentalSpatial>
-    <DefineConstants Condition="'$(UseAzureCoreExperimentalSpatial)' == 'true'">EXPERIMENTAL_SPATIAL;$(DefineConstants)</DefineConstants>
+    <UseAzureCoreExperimentalSpatial Condition="'$(UseAzureCoreExperimentalSpatial)' == ''">true</UseAzureCoreExperimentalSpatial>
```

Comment:
You could probably just remove this whole section. IIRC, it was used only to update `DefineConstants`. If you're taking a hard dependency on the package, anywhere this might be used that I'm not remembering, just make it non-conditional. You've effectively already done that by removing the preproc conditions anyway.

Reply:
This macro is further used along with `UseAzureCoreExperimentalDynamic` to determine whether to `AddExperimentalReference`.
This change was the smallest delta I could make to keep everything else behave the same way.

Reply:
We should remove all of these.  Dynamic has been decided and we're not doing it and you just fixed Spatial.